app/helpers/application_helper.rb

```diff
@@ -76,6 +76,38 @@ def format_datetime(date, format = :long)
   def time_period_class(item)
     return 'not-yet-valid' if item.not_yet_valid?
     return 'currently-valid' if item.currently_valid?
-    return 'expired' if item.expired?
+    'expired' if item.expired?
+  end
+
+  # Generates a page header. The title shown will be the +.header+ key in the page
+  # that calls this helper.
+  # @param [Class] resource_class The resource header buttons will manage. If unspecified,
+  #   the other options will be ignored.
+  # @yield A block in which other helper methods may be called, the results of which will
+  #   be placed on the far right of the header. Results of multiple helpers should be concatenated
+  #   with +:++.
+  # @return [String]
+  def page_header(resource_class = nil, &block)
+    content_tag(:div, class: 'page-header') do
+      content_tag(:h1) do
+        content_tag(:span, t('.header')) + page_header_resource_button(resource_class, &block)
+      end
+    end
+  end
+
+  # Generates a button associated with a resource. Child content will only be shown if the current
+  # user has +create+ privileges. Parameters are the same as those for +page_header+.
+  # @param [Class] resource_class
+  # @yield A block in which other helpers may be called.
+  # @see #page_header
+  # @return [String]
+  def page_header_resource_button(resource_class = nil, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.